OrbitClientGgp/ClientGgp.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CLIENT_GGP_CLIENT_GGP_H_
+#define ORBIT_CLIENT_GGP_CLIENT_GGP_H_
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+#include "ClientGgpOptions.h"
+#include "OrbitCaptureClient/CaptureClient.h"
+#include "OrbitCaptureClient/CaptureListener.h"
+#include "OrbitProcess.h"  // remove if including Capture.h in here
```

Comment:
Is this comment outdated? At least I don't get it.

Reply:
It was a note to myself when I was figuring out what it was needed to include and completely forgot to remove it, will update it, thanks!